src/test/java/com/jcabi/s3/AwsBucketITCase.java

```diff
@@ -147,4 +150,17 @@ public void listsInPrefixedBucketWithouCollisions() throws Exception {
         }
     }
 
+    /**
+     * AwsBucket can check if the bucket exists.
+     * @throws IOException If fails
+     */
+    @Test
+    public void checkBucketExistence() throws IOException {
+        final Bucket bucket = this.rule.bucket();
+        Assert.assertTrue(bucket.exists());
```

Comment:
@yegor256 ok, I see. But the implementation of the method `this.rule.bucket()` is located only in `BucketRule` class and creates nothing (returns `this.subj`). The actual bucket seems to be created with `BucketRule.create()` and dropped with `BucketRule.drop()` in finally block after test completion within the method `BucketRule.apply(Statement, Description)`. If I understand statements / rules in junit correctly. it does the same what you want, doesn't it?